sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -10,7 +10,7 @@ import { PartitionProcessor } from "./partitionProcessor";
 import { EventHubConsumer } from "./receiver";
 import { AbortController } from "@azure/abort-controller";
```

Comment:
Looks like we're not using abortSignal here. Can we narrow the type down to the field you want?

```
Pick<OperationOptions, 'tracingOptions'> // or something of that nature?
```

Reply:
Sorry, I used the vscode extension and it put the common at the top. I'm talking about the `createProcessingSpan()` signature change.

Reply:
You mean `createProcessingSpan()` doesnt need abort signal?

Reply:
yes.

Reply:
Logged #8407 for a follow up that will cover the above